templates/cluster-template-powervs-external-cloud-provider.yaml

```diff
@@ -0,0 +1,496 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
we have only external cloud provider support, so wondering if we can just keep the filename as `cluster-template-powervs-cloud-provider.yaml`

Reply:
Update it.